tests/testthat/test_agg.R

```diff
@@ -515,6 +515,49 @@ test_that(description, {
   expect_equal(output_dt, expected_dt)
 })
 
+# Include NA in interval mapping ------------------------------------------
+
+id_cols <- c("group", "age_start", "age_end")
+value_cols <- "value"
+
+input_dt <- data.table(
+  group = c(1, 1, 1, 2, 2, 2),
+  age_start = c(0, 1, NA, 0, 1, NA),
+  age_end = c(1, 2, NA, 1, 2, NA),
+  value = 1
+)
+setkeyv(input_dt, id_cols)
+
+expected_dt <- data.table(
+  group = c(1, 2),
+  age_start = c(0, 0),
+  age_end = c(2, 2),
+  value = 2
+)
```

Comment:
So if I'm understanding this correctly it will just exclude the values for NA age groups?
If there were an NA in the middle of an age pattern e.g. missing ages 1-3 in a 0-5 aggregation would it just output 0-2 and 4-5 age groups and treat NA age values as zeros?

Reply:
If so, this is a little different from what I was doing. If NA ages have values I think they're included in my sum across all ages.

Reply:
> If there were an NA in the middle of an age pattern e.g. missing ages 1-3 in a 0-5 aggregation would it just output 0-2 and 4-5 age groups and treat NA age values as zeros?

I think this scenario you described is different, it would be flagged as a "missing" age group and error out by default. This corresponds to the `missing_dt_severity` arguments

Reply:
> If so, this is a little different from what I was doing. If NA ages have values I think they're included in my sum across all ages.

The difference between line 545 and 555 is what would trigger the NA ages values to be included.
```
    mapping = data.table(age_start = 0, age_end = 2)
    mapping = data.table(age_start = 0, age_end = 2, include_NA = TRUE)
```